keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/Exponential.java

```diff
@@ -34,7 +34,7 @@ public DoubleTensor logProb(DoubleTensor x) {
 
     @Override
     public Diffs dLogProb(DoubleTensor x) {
-        final DoubleTensor dLogPdx = lambda.reciprocal().unaryMinusInPlace();
+        final DoubleTensor dLogPdx = DoubleTensor.zeros(x.getShape()).plusInPlace(lambda).reciprocalInPlace().unaryMinusInPlace();
```

Comment:
Can we just do `minusInPlace` so we avoid calling `unaryMinusInPlace`?

Reply:
Thanks!

Yep we can - I'd just changed the zeros stuff so we got the Tensor created, but looking at the equation, yeah just doing a minus is better - good catch.